symphony/Symphony/Plugins/GetWindowsPlugin.cs

```diff
@@ -1,65 +1,79 @@
-﻿using Paragon.Plugins;
-using System.Windows;
-using System;
-
-namespace Symphony.Plugins
-{
-    [JavaScriptPlugin(Name = "cef", IsBrowserSide = true)]
-    public class GetWindowsPlugin : IParagonPlugin
-    {
-        private IApplication application;
-
-        public void Initialize(IApplication application)
-        {
-            this.application = application;
-        }
-
-        public void Shutdown()
-        {
-        }
-
-        [JavaScriptPluginMember(Name = "getScreenMedia")]
-        public void cefGetScreenMedia(JavaScriptPluginCallback callback)
-        {
-            var mainWindow = (Window)this.application.WindowManager.AllWindows[0];
-
-            var showDialog = new Action(() =>
-            {
-                var window = new MediaStreamPicker.MediaStreamPicker();
-                MediaStreamPicker.MediaStreamPickerViewModel vm = new MediaStreamPicker.MediaStreamPickerViewModel(window);
-                window.DataContext = vm;
-
-                EventHandler requestCloseHandler = (sender, args) =>
-                {
-                    callback("permission-denied", null);
-                    window.Close();
-                };
-                vm.RequestCancel += requestCloseHandler;
-
-                EventHandler<MediaStreamPicker.RequestShareEventArgs> requestShareHandler = (sender, args) =>
-                {
+﻿using Paragon.Plugins;
+using System.Windows;
+using System;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Symphony.Plugins
+{
+    [JavaScriptPlugin(Name = "cef", IsBrowserSide = true)]
+    public class GetWindowsPlugin : IParagonPlugin
+    {
+        private IApplication application;
+
+        public void Initialize(IApplication application)
+        {
+            this.application = application;
+        }
+
+        public void Shutdown()
+        {
+        }
+
+        private class Sources
+        {
+            public string[] sources { get; set; }
+        }
+
+        [JavaScriptPluginMember(Name = "getScreenMedia")]
+        public void cefGetScreenMedia(JObject arguments, JavaScriptPluginCallback callback)

```

Comment:
@lneir , I´ve reviewed and applied some changes to prevent not having sources parameter.